### ISSUE TITLE
Current address layout is not compatible with screen-readers.

### ISSUE THREAD

**Issue by lat9** [lat9/edit_orders#175]
See [this](https://www.zen-cart.com/showthread.php?200407-Edit-Orders-v4-0-Support-Thread&p=1374069#post1374069) Zen Cart support-thread posting for additional information.

**Comment by jage9** [comment]
I see the formatting change has been made. Is there still a desire to add additional rows?

**Comment by lat9** [comment]
There's still the need (as present in the now-reformatted layout) for other plugins to add elements to the various address blocks, but I'd appreciate your insight as to whether the current reformatting addresses/corrects this issue.

**Comment by jage9** [comment]
Yes it's much better. Just submitted a PR to add a chunk of labels which 
helps when navigating. Will need to do this for other areas as well.

**Comment by lat9** [comment]
Closing this since EO-5's approach to the address layout of the screen is quite different than EO-4's.
